trackalert/trackalert-lua.cc

```diff
@@ -435,6 +435,13 @@ vector<std::function<void(void)>> setupLua(bool client, bool multi_lua,
     c_lua.writeFunction("setNumWebHookThreads", [](unsigned int num_threads) { });
   }
 
+  if (!multi_lua) {
+    c_lua.writeFunction("setWebHookQueueSize", [](unsigned int queue_size) { g_webhook_runner.setMaxQueueSize(queue_size); });
+  }
+  else {
+    c_lua.writeFunction("setWebHookQueueSize", [](unsigned int queue_size) { });
```

Comment:
maybe this could emit warning?

Reply:
Warning would be bad here, as this will intentionally be called for every lua state, but it only "makes sense" for one particular lua state (same goes for the other similar examples)